src/Components/AdminPortal/index.js

```diff
@@ -0,0 +1,216 @@
+import React from "react";
+import Cookies from "js-cookie";
+
+export default class AdminPortal extends React.Component {
+  constructor(props) {
+    super(props) 
+
+    this.state = {
+      projectCreated: false,
+    }
+  }
+
+  creationSuccess = async id => {
+    await this.setState({
+      projectCreated: true,
+      id: id
+    })
+  };
+
+  creationFailure = () =>  {};
+
+  onFieldChange = (value, field) => {
+    this.setState({
+      [field]: value
+    })
+  }
+
+  userAddedSuccess = async () => {
+    this.setState({
+      userAdded: true
+    });
+  };
+
+  userAddedFailure = () => {};
+
+  onProjectCreate = async () => {
+    if(this.state.name && this.state.type) {
+      await this.props.createProject.execute(this, this.state.name, this.state.type)
+
+      await this.addUser()
+      Cookies.remove('apikey');
+      window.location.reload();
+    }
+  }
+  
+  addUser = async () => {
+    await this.props.addUsersToProject.execute(this, this.state.id,[{email: this.state.email, role: this.state.role}])
+  }
+
+  renderCreateProject= () => {
+    return <div>
+        <h3>
+          Create a new project
+        </h3>
+        <p>You will need to login again after creating the project</p>
+        {this.renderProjectForm()}
+      </div>
+  }
+
+  renderSuccessMessage = () => {
+    if (this.state.projectCreated) {
+      return <p data-test="project-created-message">Your new project has been created!</p>
+    } else if (this.state.userAdded){
+      return <p data-test="user-added">The user has been added!</p>;
+    } else {
+      return null;
+    }
+  } 
+  
+  renderProjectDetails = () => {
+    return <div>
+      <div className="form-group">
+        <label htmlFor="projectName">Enter the name of the project</label>
+        <input
+          className="form-control"
+          id="projectName"
+          data-test="create-project-name"
+          placeholder="Project Name"
+          onChange={e => this.onFieldChange(e.target.value, "name")}
+        />
+      </div>
+      <div className="form-group">
+        <label htmlFor="projectType">Please select the scheme for this project</label>
+        <select
+          data-test="create-project-type"
+          className="form-control"
+          id="projectType"
+          onChange={e => this.onFieldChange(e.target.value, "type")}
+        >
+          <option value="ac">Accelerated Construction</option>
+          <option value="hif">Marginal Viability Fund</option>
+          
+        </select>
+      </div>
+    </div>
+  }
+
+  renderUserDetails = () => {
+    return <div>
+      <div className="form-group">
+        <label htmlFor="userEmail">Enter the users email here.</label>
```

Comment:
```suggestion
        <label htmlFor="userEmail">Enter the user's email here.</label>
```
Not sure if the apostrophe needs to be escaped as it's in a tag?